CS.Security/Controllers/AdminsController.cs

```diff
@@ -10,35 +10,37 @@ namespace CS.Security.Controllers;
 [Authorize(Roles = "SuperAdmin")]
 public class AdminsController : ControllerBase
 {
-    private readonly IAdminService _adminService;
+    private readonly IUserService _userService;
```

Comment:
In general, both AdminService and UserService share a lot of common code. I found it reasonable to use a single one, but define the behavior outside. Admin service seem to not have anything Admin-specific, so code reduction might be useful here